frank-doc-doclet/src/main/java/org/frankframework/frankdoc/model/FrankElement.java

```diff
@@ -238,9 +238,9 @@ private void assembleParsedJavaDocTags(FrankClass clazz, String tagName, Consume
 	}
 
 	private void handleLabels(FrankClass clazz, LabelValues labelValues) {
-		List<FrankAnnotation> annotationsForLabels = Arrays.asList(clazz.getAnnotations()).stream()
+		List<FrankAnnotation> annotationsForLabels = Arrays.stream(clazz.getAnnotations())
 			.filter(a -> a.getAnnotation(LABEL) != null)
-			.collect(Collectors.toList());
+			.toList();
 		for (FrankAnnotation a : annotationsForLabels) {
```

Comment:
Staat los van jouw change hoor, maar ik zou het mooi vinden als deze for dan ook gelijk in de lambda meekan. Dat gaat nu niet doordat `handleSpecificLabel` een exceptie gooit, maar volgens mij kan je de try/catch in die method opnemen waardoor je ipv `.toList()` de loop gelijk kan afhandelen in de lambda: `forEach(annotation -> handleSpecificLabel(annotation, labelValues)`.

Reply:
Die exceptie is ook niet meer nodig door een andere veranding in deze PR. Goeie spot, ik zal het aanpassen.